index.html

```diff
@@ -267,11 +267,8 @@ <h2>Contributing</h2>
       this specification:</p>
 
     <ul>
-      <li>Technical discussion typically occurs on the public mailing list:
-        <a href="https://lists.w3.org/Archives/Public/public-linked-json/">public-linked-json@w3.org</a></li>
-
-      <!--<li><a href="https://json-ld.org/minutes/">Public teleconferences</a> are held
-        on Tuesdays at 1500UTC on the second and fourth week of each month.</li> -->
+      <li>Technical discussion typically occurs on the working group mailing list:
+        <a href="https://lists.w3.org/Archives/Public/public-json-ld-wg/">public-json-ld-wg@w3.org</a></li>
 
       <li>The <a href="https://webchat.freenode.net/?channels=json-ld">#json-ld</a>
         IRC channel is available for real-time discussion on irc.freenode.net.</li>
```

Comment:
Should update this to the w3c wg channel or perhaps list both?

Reply:
I'd say list both.

Reply:
@iherman is there a URL to reference the #json-ld channel on irc.w3.org directly?